### ISSUE TITLE
template option for form

### ISSUE THREAD

**Issue by ahmadazimi** [kristijanhusak/laravel-form-builder#132]
Hi there,

Is there any way to pass custom template for a form. I'm going to create some templates for different form styles (for example vertical, horizontal and ...)

Is below code possible?

```php
$form = $this->plain([
    'url' => '/',
    'method' => 'POST',
    'template' => 'w3g::form-builder.form'
])->add('email', 'email', [
    'label' => 'Email address'
]);
```

**Comment by kristijanhusak** [comment]
Hi,

No, that's not possible at this moment, but I will probably add it soon. Stay tuned.

**Comment by ahmadazimi** [comment]
Thank you.
I'm waiting for that ;-)

**Comment by kristijanhusak** [comment]
This is now available on master branch.

**Comment by ahmadazimi** [comment]
Thank you :+1:

**Comment by kristijanhusak** [comment]
Available on latest version (1.6.20). Thanks!

**Comment by ghprod** [comment]
thats greaaatt :D

Thanks Kris
